### PR TITLE
Add as_blender_areaLight shader

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: src/appleseed.shaders/src/blenderseed/as_blender_areaLight.osl [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
+//
+// This source file is part of appleseed.
+// Visit https://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2016-2018 Luis Barrancos, The appleseedhq Organization
```
Comment (dictoon) [inline src/appleseed.shaders/src/blenderseed/as_blender_areaLight.osl]: Fix copyright notice.

File: src/appleseed.shaders/src/blenderseed/as_blender_areaLight.osl [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        string widget = "checkBox",
+        int as_blender_input_socket = 0
+    ]],
+    output closure color out_output = 0
+    [[
+        string label = "Output"
+    ]]
+)
+{
+
```
Comment (dictoon) [inline src/appleseed.shaders/src/blenderseed/as_blender_areaLight.osl]: Remove extra blank line.

**Comment by dictoon** [comment]
What's the purpose of this shader?

**Comment by jdent02** [comment]
It’s for area lights in blender. Esteban and I had a pretty in depth conversation in the Blender channel about it yesterday

**Comment by jdent02** [comment]
Oh, I put in your suggestions.

File: src/appleseed.shaders/CMakeLists.txt [hunk 3]
```diff
@@ -156,6 +156,7 @@ source_group ("include\\appleseed\\transform" FILES
 
 set (src_blenderseed_sources
      src/blenderseed/as_closure2surface.osl
+     src/blenderseed/as_blender_areaLight.osl
```
Comment (dictoon) [inline src/appleseed.shaders/CMakeLists.txt]: One last thing: can you sort this alphabetically? You can merge afterward. Thanks!

Comment (jdent02) [reply src/appleseed.shaders/CMakeLists.txt]: Will do.  I don't have merge rights on this repo :)

Comment (dictoon) [reply src/appleseed.shaders/CMakeLists.txt]: What?? Wait :)

**Pull request merged**